src/period.cpp

```diff
@@ -8,6 +8,15 @@
 #include "nanotime/pseudovector.hpp"
 #include "nanotime/utilities.hpp"
 
+// See comment in <R_Ext/Complex.h> in R>=4.3.0 and #134. Old
+//   approach to do `Rcomplex{ re, im }` now gives compiler issue
+//   on some compilers, R versions.
```

Comment:
I would, if we could, like to edit the comment here somewhat at stress that 'modern' compilers and 'modern' C++ instantation get into each others ways on the _dual_ definition of the type, and that reverting to very plain C like assignment within the `struct` works.

Reply:
OK, I proposed some phrasing as a suggested change. Let's edit that comment to satisfaction & then commit.